templates/.snapshots/TestRenderAFile-.circleci-config.yml.tpl-.circleci-config.yml.snapshot

```diff
@@ -2,7 +2,7 @@
 # syntax, such as anchors, will be fixed automatically.
 version: 2.1
 orbs:
-  shared: getoutreach/shared@dev:cache
+  shared: getoutreach/shared@dev:
```

Comment:
does this resolve latest?

Reply:
The module doesn't resolve at all in `stenciltest`, which is why it is empty. @artemchernyak-outreach is currently fixing that.

Reply:
ic

Reply:
In normal operation, I tested all of these modes on `orc`, it would resolve the latest based on the version resolver criteria.